Gemfile

```diff
@@ -6,3 +6,7 @@ group :development do
   gem 'smart_proxy', :git => "https://github.com/theforeman/smart-proxy", :branch => "develop"
   gem 'pry'
 end
+
+# load local gemfile
+local_gemfile = File.join(File.dirname(__FILE__), 'Gemfile.local')
```

Comment:
bikesheding: `Gemfile.local.rb` would be better, as editors recognize it's a ruby source.

Also adding it to `.gitignore` would make sense here